codec/src/main/java/io/netty/handler/codec/compression/Lz4FrameEncoder.java

```diff
@@ -341,4 +370,12 @@ public void handlerRemoved(ChannelHandlerContext ctx) throws Exception {
         super.handlerRemoved(ctx);
         cleanup();
     }
+
+    public int getBlockSize() {
```

Comment:
actually I think we should also remove the `get` prefix as we not have it in `currentBlockLenght` as well.